### PR TITLE
Add build, rename class operators

### PR DESCRIPTION
**Pull request by garyb**
Thoughts on these operator names?

I also removed `/=` from the `Eq` class, we may as well considering it's implemented as `not (x == y)` _everywhere_ that I know of - we can always reintroduce it when default class members are allowed.

### CODE REVIEW

**Comment by paf31** [comment]
Looks great. Do the optimizations all still work?

**Comment by garyb** [comment]
I haven't tested anything like that yet, was more opening this for discussion. :)

**Comment by paf31** [comment]
Sure :) I like it ...

**Comment by garyb** [comment]
Any comments from @jdegoes, @joneshf, @puffnfresh?

File: docs/Prelude.md [hunk 1]
```diff
@@ -0,0 +1,1185 @@
...
+[`Applicative`](#applicative) functor, without using `(<$>)` as provided
+by the [`Functor`](#functor)-[`Applicative`](#applicative) superclass
+relationship.
+
+`liftA1` can therefore be used to write [`Functor`](#functor) instances
+as follows:
+
+```purescript
+instance functorF :: Functor F where
+  (<$>) = liftA1
```
Comment (paf31) [inline docs/Prelude.md]: `map`

**Comment by puffnfresh** [comment]
:+1: to the derived `/=`.

**Comment by jdegoes** [comment]
:+1:

**Pull request merged**